examples/cpu/CMakeLists.txt

```diff
@@ -6,3 +6,10 @@ target_link_libraries (seq_example LINK_PUBLIC traccc::core traccc::io vecmem::c
 
 add_executable (ccl_example ccl_example.cpp)
 target_link_libraries (ccl_example LINK_PUBLIC traccc::core traccc::io)
+
+# find OpenMP
+find_package(OpenMP REQUIRED)
+set(GCC_OMP "-fopenmp")
+set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} ${GCC_OMP}")
+add_executable (par_example par_example.cpp)
+target_link_libraries (par_example LINK_PUBLIC traccc::core traccc::io vecmem::core OpenMP::OpenMP_CXX)
```

Comment:
Perhaps we can make OpenMP optional? That is to say, check if it's there, and compile the OpenMP examples only then.

Reply:
Rebased & implemented the required changes.